android/app/src/main/AndroidManifest.xml

```diff
@@ -19,7 +19,7 @@
          FlutterApplication and put your custom class here. -->
     <application
         android:name=".MultidexApplication"
-        android:label="Natrium"
+        android:label="Nyano"
```

Comment:
We're obviously still debating this, but ideally wouldn't this be "Nyano Wallet" rather than just "Nyano"?

Reply:
Going to go with Nyano Mobile for the time being to keep it clear. I quickly pulled together a Nyano Mobile theme in pink but its pretty ghastly haha